resources/public/api/conf/1.0/application.yaml

```diff
@@ -0,0 +1,151 @@
+openapi: 3.0.0
+info:
+  title: National Insurance Test Support
+  contact: {}
+  version: '1.0'
+servers:
+- url: https://api.service.hmrc.gov.uk/

```

Comment:
Don't think this is correct. This is a production url and from [here](https://catalogue.tax.service.gov.uk/whats-running-where?profile_type=team&application_name=national-insurance-des-stub&profile_name=DDCE+Live+Service&view_mode=versions) the screenshot, it says it is not available in production
<img width="662" alt="Screenshot 2022-12-15 at 12 59 00" src="https://user-images.githubusercontent.com/63714619/207865105-9c06d564-b080-4a7d-8d16-689d74198122.png">

I believe only `https://test-api.service.hmrc.gov.uk` is applicable here

For the APIs we have migrated for example the screenshot below, this would be correct to add.

<img width="671" alt="Screenshot 2022-12-15 at 13 03 37" src="https://user-images.githubusercontent.com/63714619/207866017-0fb7d1c1-7f11-4db6-a9c6-402fdd169c1d.png">